core/src/main/java/org/mskcc/cbio/portal/util/CancerStudyPermissionEvaluator.java

```diff
@@ -205,11 +205,13 @@ private boolean hasPermission(CancerStudy cancerStudy, Authentication authentica
         if (GlobalProperties.filterGroupsByAppName()) {
             for (String au : allAuthorities) {
                 if (au.toUpperCase().startsWith(appName + ":")) {
-                    grantedAuthorities.add(au.substring(appName.length() + 1));
+                    grantedAuthorities.add(au.substring(appName.length() + 1).toUpperCase());
```

Comment:
@n1zea144 It actually improves it since it makes the process case insensitive. See row 186 in this file: 
```java
boolean toReturn = grantedAuthorities.contains(stableStudyID.toUpperCase());
```
this checks the authorities against the `stableStudyID.toUpperCase()`. Previously this meant that all authorities in the DB table had to be stored in UPPER CASE (see also old documentation in `User-Authorization.md` changed below). By setting `grantedAuthorities.add(au.substring(appName.length() + 1).toUpperCase())` here, I removed this requirement, making it easier to fill in the authorities table.